### DISCUSSION TITLE
Object Align

### DISCUSSION THREAD

**Discussion by Sunoctagram** [PintaProject/Pinta#813] [Ideas]
A simple object align tool to center a layers content. Ive been using a Paint.net plugin that adds this and i cannot live without it. This is the one thing that is pushing from fully switching to Pinta
![Untitled](https://github.com/PintaProject/Pinta/assets/170787081/5abd59ee-b7b4-4487-8ec5-43ecb8cea1c7)

**Comment by cameronwhite** [comment]
This would be a good feature for someone to contribute 👍  It's probably not too hard and could reuse lots of code from the Move Selected Pixels and Auto Crop (bounding rectangle) tools

**Comment by Gingermanuk** [comment]
I too would love to see this added as part of the build or to allow to be used as a plugin.  I moved from windows as my main at the beginning of last year and paint.net and one other app is all what is missing for me to finally stop dual booting and have Linux as my main and only OS. I dont need all the other fancy plugins but object align tool is something that I use a lot and would love to see in Pinta.

**Comment by snoozysoft** [comment]
this + drop shadow are the two third party plugins i use a lot for paint net so yeah this would be absolutely ideal if it was in pinta by default. only just now found out about this program and im already very pleased with what ive seen.

**Comment by cameronwhite** [comment]
I've transferred this to issue #936
